members-list.md

```diff
@@ -16,7 +16,10 @@ Also don't be a cop-out by saying something like "I am happy/looking forward to
 **Do not modify anything above this line**  
 
 ## The list 
-[Add your details here]`<--Don't remove this line`<br/>
```

Comment:
Welcome!

Please do not remove this line xD, it's required for newer members to know where to add their details!